src/utils/validators/index.js

```diff
@@ -53,6 +71,26 @@ const InputValidators = {
       }
     },
 
+    /**
+     * @since 3.10.0
+     * @param options
+     */
+    chart(options) {
+      this.options = options;
+      this.validate = function(value) {
+        return value && 1 === `${value}`.length;
+      }
+    },
```

Comment:
@volterra79 is this an oversight? ie. `chart(options)` → `char(options)`

Reply:
@Raruto rigth. It's a misspelling